public/app/plugins/datasource/cloud-monitoring/components/InlineFields.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react';
```

Comment:
I guess this super label component could potentially by a candidate for grafana ui. But really, there's not that much that it does...thoughts?